README.md

```diff
@@ -13,7 +13,16 @@ Bevy is a refreshingly simple data-driven game engine built in Rust. It is free
 
 ## WARNING
 
-Bevy is still in the _very_ early stages of development. APIs can and will change (now is the time to make suggestions!). Important features are missing. Documentation is sparse. Please don't build any serious projects in Bevy unless you are prepared to be broken by API changes constantly.
+Bevy is still in the early stages of development.
+APIs can and will change in both trivial and critical ways.
+Important features are incomplete.
+Documentation can be spotty.
+
+If you need something, please feel free to [help out](CONTRIBUTING.md)!
+We're always looking for contributors, and your hands-on experience with the engine is invaluable when deciding how something should work.
```

Comment:
I wonder if we can almost cut this entirely.

Maybe have a note about being friendly to contribution in the very first section?

"If you need" isn't great language, since it suggests not merging a PR is "not meeting a **need**", which isn't great.